<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>883</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>991</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,20">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QFrame" name="ObjectFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
QFrame {
	background-color: rgb(236, 236, 236);
   border: 2px solid lightgrey;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>678</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="ObjectLayout">
         <item>
          <widget class="QPushButton" name="Host">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/host.png</normaloff>../Icon/host.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="STA">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/sta.png</normaloff>../Icon/sta.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UE">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/ue.png</normaloff>../Icon/ue.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GNB">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/gNB.png</normaloff>../Icon/gNB.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DockerHost">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/docker.png</normaloff>../Icon/docker.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AP">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/AP.png</normaloff>../Icon/AP.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="VGcore">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/5G core.png</normaloff>../Icon/5G core.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Router">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/Router.png</normaloff>../Icon/Router.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Switch">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/switch.png</normaloff>../Icon/switch.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="LinkCable">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/link cable.png</normaloff>../Icon/link cable.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Controller">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Icon/controller.png</normaloff>../Icon/controller.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="shortcut">
            <string>H</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Canvas" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>1828</width>
         <height>800</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>1280</width>
         <height>720</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport_to_Level_2_Script"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionPreference"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_NetFlux5G"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPickTool"/>
   <addaction name="actionHandTool"/>
   <addaction name="actionDelete"/>
   <addaction name="actionTextBox"/>
   <addaction name="actionDrawSquare"/>
   <addaction name="separator"/>
   <addaction name="actionShowGrid"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionResetZoom"/>
   <addaction name="separator"/>
   <addaction name="actionRunAll"/>
   <addaction name="actionStopAll"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_to_Level_2_Script">
   <property name="text">
    <string>Export to Level 2 Script</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preference</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout_NetFlux5G">
   <property name="text">
    <string>About NetFlux5G</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPickTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/cursor.webp</normaloff>../Icon/cursor.webp</iconset>
   </property>
   <property name="text">
    <string>Pick Tool</string>
   </property>
  </action>
  <action name="actionHandTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/handTool.png</normaloff>../Icon/handTool.png</iconset>
   </property>
   <property name="text">
    <string>Hand Tool</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/delete.png</normaloff>../Icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionTextBox">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/textBox.png</normaloff>../Icon/textBox.png</iconset>
   </property>
   <property name="text">
    <string>Text Box</string>
   </property>
  </action>
  <action name="actionDrawSquare">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/drawSquare.png</normaloff>../Icon/drawSquare.png</iconset>
   </property>
   <property name="text">
    <string>Draw Square</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/grid.png</normaloff>../Icon/grid.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/zoomIn.png</normaloff>../Icon/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/zoomOut.png</normaloff>../Icon/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/resetZoom.png</normaloff>../Icon/resetZoom.png</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionRunAll">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/RunAll.png</normaloff>../Icon/RunAll.png</iconset>
   </property>
   <property name="text">
    <string>RunAll</string>
   </property>
  </action>
  <action name="actionStopAll">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/StopAll.png</normaloff>../Icon/StopAll.png</iconset>
   </property>
   <property name="text">
    <string>StopAll</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
